license.md

```diff
@@ -1,3 +1,5 @@
+Revised BSD License
+
 Copyright (c) 2016, Damian Mooyman
```

Comment:
Not important, but could be 2017 ;)

Reply:
I'd avoided doing this recently because it'll have to be updated again in a couple of weeks anyway =D

Reply:
Yeah, super unimportant :P